### PR TITLE
fix doc bug

### PR DESCRIPTION
**Pull request by MyXOF**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -187,10 +187,12 @@ Now let's trying to read and write some data from IoTDB using our Client. To sta
 cd cli/cli
 
 # Unix/OS X
-> $IOTDB_HOME/bin/start-client.sh -h <IP> -p <PORT> -u <USER_NAME>
+> cd /workspace/incubator-iotdb/iotdb-cli/cli/bin/
+> start-client.sh -h <IP> -p <PORT> -u <USER_NAME>
```
Comment (jixuan1989) [inline Readme.md]: `./start-client.sh` is correct...
By the way, I suggest write the document as:
```
suppose your iotdb-cli is on $IOTDB_CLI_HOME, e.g. `export IOTDB_CLI_HOME = /workspace/incubator-iotdb/iotdb-cli`
> $IOTDB_CLI_HOME/bin/start-client.sh
```

File: Readme.md [hunk 2]
```diff
@@ -187,10 +187,12 @@ Now let's trying to read and write some data from IoTDB using our Client. To sta
...
 
 # Windows
-> $IOTDB_HOME\bin\start-client.bat -h <IP> -p <PORT> -u <USER_NAME>
+> cd \workspace\incubator-iotdb\iotdb-cli\cli\bin
+> start-client.bat -h <IP> -p <PORT> -u <USER_NAME>
```
Comment (jixuan1989) [inline Readme.md]: I am not sure whether `./` is needed at the head of `start-client.bat` on Win OS.

File: docs/Documentation/Frequently asked questions.md [hunk 3]
```diff
@@ -122,5 +138,5 @@ IoTDB> show timeseries root.fit.d1.*
 The default IoTDB's CLI time display format is human readable (e.g. ```1970-01-01T08:00:00.001```), if you want to display time in timestamp type or other readable format, add parameter ```-disableIS08601``` in start command:
 
 ```
->sh IOTDB_HOME/iotdb/bin/start-client.sh -h 10.129.187.21 -p 6667 -u root -pw root -disableIS08601
-```
\ No newline at end of file
+> sh /workspace/incubator-iotdb/iotdb-cli/cli/bin/start-client.sh -h 127.0.0.1 -p 6667 -u root -pw root -disableIS08601
```
Comment (jixuan1989) [inline docs/Documentation/Frequently asked questions.md]: This is better than `cd /workspace/incubator-iotdb/iotdb-cli/cli/bin && ./start-client.sh ...`

**Comment by qiaojialin** [comment]
The cli/lib should also be removed:

```
> rm -rf $IOTDB_HOME/data/
> rm -rf $IOTDB_HOME/lib/
```

The iotdb-cli model also need to be build, then it will generate jars in the cli/lb. So the following command may need to be replaced by:  mvn clean package -DskipTests

```
> mvn clean package -pl iotdb -am -Dmaven.test.skip=true
```

**Comment by MyXOF** [comment]
image
When you build cli project, it will clean files under libs directoru

**Comment by jixuan1989** [comment]
> The cli/lib should also be removed:
> 
> ```
> > rm -rf $IOTDB_HOME/data/
> > rm -rf $IOTDB_HOME/lib/
> ```
> The iotdb-cli model also need to be build, then it will generate jars in the cli/lb. So the following command may need to be replaced by: mvn clean package -DskipTests
> 
> ```
> > mvn clean package -pl iotdb -am -Dmaven.test.skip=true
> ```

why part of document you are discussing?

**Comment by qiaojialin** [comment]
> > The cli/lib should also be removed:
> > ```
> > > rm -rf $IOTDB_HOME/data/
> > > rm -rf $IOTDB_HOME/lib/
> > ```
> > The iotdb-cli model also need to be build, then it will generate jars in the cli/lb. So the following command may need to be replaced by: mvn clean package -DskipTests
> > ```
> > > mvn clean package -pl iotdb -am -Dmaven.test.skip=true
> > ```
> 
> why part of document you are discussing?

QuickStart.md

**Comment by jixuan1989** [comment]
> The iotdb-cli model also need to be build, then it will generate jars in the cli/lb. So the following command may need to be replaced by: mvn clean package -DskipTests
> > mvn clean package -pl iotdb -am -Dmaven.test.skip=true

@qiaojialin 
Either `mvn clean package -DskipTests` and `mvn clean package -pl iotdb -am -Dmaven.test.skip=true` is OK. But the former is more simply.

> The cli/lib should also be removed:
>> rm -rf $IOTDB_HOME/data/
>> rm -rf $IOTDB_HOME/lib/
Jialin is right, @MyXOF , maven does not delete the data folder.

**Comment by MyXOF** [comment]
I add how to build client project
image

**Comment by jixuan1989** [comment]
please squash and merge

**Pull request merged**